food-truck-api/src/main/java/food/truck/api/truck/TruckService.java

```diff
@@ -49,17 +49,13 @@
         return truckRepository.findById(truckId);
     }
 
-    public Truck saveTruck(Truck truck) {
-        return truckRepository.save(truck);
-    }
-
     public Truck createTruck(Long userId, String name) {
         var t = new Truck();
         t.setName(name);
         t.setUserId(userId);
         t.setDescription("");
         t.setTags(new HashSet<>());
-        return saveTruck(t);
+        return truckRepository.save(t);
     }
```

Comment:
Are we getting rid of the `saveTruck` method? I'll need to update that in my PR as well if so.